src/morgenstern.jl

```diff
@@ -111,14 +314,61 @@ struct AllPairs <: MorgensternAlgorithm end
 struct FirstOnly <: MorgensternAlgorithm end
 
 """
-Create alternative Morgenstern generators using all pairwise products (i≠j).
```

Comment:
the added documentation is very illuminating, but removing this line kinda makes it hard to know **what** this function is, even if the added docs give a ton of background for how things are implemented